DOCUMENTATION.md

```diff
@@ -400,6 +400,7 @@ If the request is not successful, the client raises a `Notifications::Client::Re
 |`400`|`BadRequestError: Cannot send letters with a team api key`|`BadRequestError`|Use the correct type of [API key](#api-keys)|
 |`400`|`BadRequestError: Cannot send letters when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode`|`BadRequestError`|Your service cannot send this notification in [trial mode](https://www.notifications.service.gov.uk/features/using-notify#trial-mode)|
 |`400`|`ValidationError: personalisation address_line_1 is a required property`|`BadRequestError`|Ensure that your template has a field for the first line of the address, refer to [personalisation](#personalisation-required) for more information|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "Must be a real UK postcode"`<br>`}]`|Ensure that the value for the postcode field in your letter is a real UK postcode|
```

Comment:
The third column here should contain the class of the error (I think it's just this client that has an extra column in the error tables)

Reply:
Good spot, thanks 💙 

That's fixed now ☺️ 
<img width="888" alt="Screenshot 2020-03-20 at 11 20 05" src="https://user-images.githubusercontent.com/20957548/77159272-ef45e880-6a9c-11ea-941e-72fc61793926.png">